### ISSUE TITLE
Not able to register an Ubuntu host where ssm agent was installed using SNAP.

### ISSUE THREAD

**Issue by jocamp18** [diodonfrost/ansible-role-amazon-ssm#5]
##### SUMMARY
Not able to register an Ubuntu host where ssm agent was installed using SNAP.

##### STEPS TO REPRODUCE
Execute the role over an Ubuntu 20.04 host with `activation_id`, `activation_code` and `aws_region` values.
```
- hosts: all
  become: yes
  roles:
  - role: diodonfrost.amazon_ssm
    vars:
      aws_region: "{{ region }}"
      activation_id: "{{ id }}"
      activation_code: "{{ code }}"
```

##### EXPECTED RESULTS
Ubuntu host is registered in AWS SSM using the ssm_agent_bin as `/snap/amazon-ssm-agent/current/amazon-ssm-agent` or `snap run amazon-ssm-agent`

##### ACTUAL RESULTS
It executes `amazon-ssm-agent` for registering the instance but as it was installed using Snap, it is not able to find the file.
```
TASK [diodonfrost.amazon_ssm : Linux | Register on-premise host] ****************************************************************************************************************************************************************************************************************
fatal: [<ip>]: FAILED! => {"changed": false, "cmd": "amazon-ssm-agent -register -code <code> -id <id> -region <region>", "msg": "[Errno 2] No such file or directory: b'amazon-ssm-agent'", "rc": 2}
```

**Comment by diodonfrost** [comment]
Thanks for your report, I'm going to investigate and write a fix when applicable.

**Comment by diodonfrost** [comment]
Fixed in the version [2.0.2](https://github.com/diodonfrost/ansible-role-amazon-ssm/releases/tag/2.0.2)
thanks for your input!

**Comment by jocamp18** [comment]
Thanks!
